scripts/create-package/fs-utils.test.ts

```diff
@@ -0,0 +1,113 @@
+import memfs, { vol } from 'memfs';
```

Comment:
Would you prefer that I convert this to the `utils` equivalent? `memfs` does seem well maintained and appears to have an extensive user base.

Reply:
Hmm. I was going to say no, this is fine, except that I went to the issues list for the repo and it seems that there are some features which are missing, for instance: https://github.com/streamich/memfs/issues/967 and https://github.com/streamich/memfs/issues/898. So that does make me pause a bit. Using a real filesystem is slower, but it would always work as we upgrade Node. So for that reason I'm inclined to use the sandbox.

Reply:
Done in e6f7274 and 8bc3c049.